DataSetExplorer/DataSetSerializer/BetweenAnnotatorsSanityCheckExporter.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using CodeModel.CaDETModel.CodeItems;
+using DataSetExplorer.DataSetBuilder.Model;
+using OfficeOpenXml;
+
+namespace DataSetExplorer.DataSetSerializer
+{
+    class BetweenAnnotatorsSanityCheckExporter
+    {
+        private readonly string _templatePath = "../../../DataSetSerializer/Template/Between_Annotators_Sanity_Check_Template.xlsx";
+        private readonly string _exportPath;
+        private ExcelPackage _excelFile;
+        private ExcelWorksheet _sheet;
+
+        public BetweenAnnotatorsSanityCheckExporter(string exportPath)
+        {
+            _exportPath = exportPath;
+        }
+
+        public void Export(List<Tuple<DataSetInstance, Dictionary<CaDETMetric, double>>> dataSetInstances,
+            string fileName, int severity)
+        {
+            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
+            _excelFile = new ExcelPackage(new FileInfo(_templatePath));
+            _sheet = _excelFile.Workbook.Worksheets[0];
+            PopulateTemplate(dataSetInstances, severity);
+            Serialize(fileName + severity);
+        }
+
+        private void PopulateTemplate(List<Tuple<DataSetInstance, Dictionary<CaDETMetric, double>>> instances, int severity)
```

Comment:
Why is severity important here?

Reply:
We run Manova test for each severity, and I generate separate files for those tests.

![image](https://user-images.githubusercontent.com/15254876/118634175-89eda480-b7d2-11eb-84f7-b715da6f5761.png)